src/columns.ts

```diff
@@ -56,7 +57,7 @@ export function findAITargetColumns(
     }
 
     // Skip if model is not one we support (case insensitive match)
-    const supportedModels = ["google", "acts2", "piglatin"];
+    const supportedModels = Object.values(TranslationModel);
```

Comment:
This is the real fix. Most of the rest is unnecessary refactor I just thought I'd do while I'm at it